core/src/main/java/io/micronaut/core/annotation/AnnotationUtil.java

```diff
@@ -45,11 +45,12 @@ public class AnnotationUtil {
             Inherited.class.getName(),
             SuppressWarnings.class.getName(),
             Override.class.getName(),
-            Repeatable.class.getName(),
+//            Repeatable.class.getName(),
```

Comment:
is this intentional?

Reply:
Somehow we have this code in the bean definition writer:
```
            final String repeatableName = visitorContext
                    .getClassElement(annotationName)
                    .flatMap(ce -> ce.stringValue(Repeatable.class)).orElse(null);
```
Not sure how it was working before.